mergekit/evo/actors.py

```diff
@@ -266,7 +266,7 @@ def evaluate(self, genotype: torch.Tensor) -> dict:
             assert (
                 model.llm_engine.parallel_config.world_size == 1
             ), "Must be single GPU"
-            worker = model.llm_engine.driver_worker
+            worker = model.llm_engine.model_executor.driver_worker
```

Comment:
Maybe:
```python
if hasattr(model.llm_engine, "driver_worker"):
    worker = model.llm_engine.driver_worker
else:
    worker = model.llm_engine.model_executor.driver_worker
```

So we can keep things working both with the pinned vllm in dependencies and more current versions.